ansible/roles/legion_airflow/tasks/install_airflow.yml

```diff
@@ -0,0 +1,61 @@
+---
+
+- name: Get airflow chart status
+  shell: helm --kube-context {{ cluster_name }} ls --all airflow-{{ enclave }}
+  register: airflow_deployment_status
+
+- set_fact:
+    airflow_deployed: "{{ airflow_deployment_status.stdout_lines|length > 1 }}"
+
+- name: Remove airflow chart
+  shell: helm --kube-context {{ cluster_name }} delete --purge airflow-{{ enclave }}
+  when: airflow_deployed
+
+- name: WORKAROUND delete pods in phase terminating
+  shell: kubectl --context {{ cluster_name }} delete --grace-period=0 --force po $(kubectl --context {{ cluster_name }} get po -o wide | grep Terminating | awk '{ print $1 }') || true
+  when: airflow_deployed
+
+- name: Wait for PVC deletion
+  shell: kubectl --context {{ cluster_name }} -n {{ enclave }} get persistentvolumeclaim
+  register: cmd_result
+  until: cmd_result.stdout.find("{{ airflow_dags_pvc }}") == -1
+  retries: 10
+  delay: 10
+
+- name: Generate fernet key
+  command: python -c 'from cryptography.fernet import Fernet; FERNET_KEY = Fernet.generate_key().decode(); print(FERNET_KEY)'
+  register: fernet_key_output
+
+- name: Generate flask key
+  command: python -c 'import os, binascii; print(binascii.b2a_hex(os.urandom(60))[:32])'
+  register: flask_key_output
+
+- name: Get Airflow Postgres RDS endpoint
```

Comment:
Looks like this piece should be in legion-aws, cause this is aws-specific

Reply:
The original point about splitting Airflow (as any other plugin) and Legion was that Legion shouldn't know anything about addons, and respectively shouldn't contain any resources provisioning/configuration/etc for them. 
We may want or not to install Airflow above the Legion so we have no reason to provision resources for Airflow with core Legion platform each time.
@dsuslov could you please clarify if we've changed the vision around that?